packages/client/src/checkout/__fixtures__/getOperation.fixture.ts

```diff
@@ -0,0 +1,21 @@
+import { rest, RestHandler } from 'msw';
+import urlJoin from 'proper-url-join';
+import type { CheckoutOrderOperation } from '../types/checkoutOrderOperation.types';
+import type { GetOperationParams } from '../types/getOperation.types';
+
+const buildUrl = ({ orderId, operationId }: GetOperationParams) =>
+  urlJoin('/api/checkout/v1/orders/', orderId, 'operations', operationId);
```

Comment:
I think you can to this like: 

```suggestion
  const URL = "/api/checkout/v1/orders/:orderId/operations/:operationId"
```

and thus avoid receiving `orderId` and `operationId` in the success/failure fixtures :)